packages/react-devtools-shared/src/utils.js

```diff
@@ -52,16 +52,30 @@ const cachedDisplayNames: WeakMap<Function, string> = new WeakMap();
 // Try to reuse the already encoded strings.
 const encodedStringCache = new LRU({max: 1000});
 
-export function alphaSortKeys(a: string, b: string): number {
-  if (a > b) {
+export function alphaSortKeys(
+  a: string | number | Symbol,
+  b: string | number | Symbol,
+): number {
+  if (a.toString() > b.toString()) {
     return 1;
-  } else if (b > a) {
+  } else if (b.toString() > a.toString()) {
     return -1;
   } else {
     return 0;
   }
 }
 
+export function getAllEnumerableKeys(
+  obj: Object,
+): Array<string | number | Symbol> {
+  const keys = [];
+  for (const key in obj) {
+    keys.push(key);
+  }
+
+  return keys.concat(Object.getOwnPropertySymbols(obj));
+}
```

Comment:
`for...in` includes inherited non-symbol keys, but `Object.getOwnPropertySymbols` does not include inherited symbol keys.

If we wanted to *not* include any inherited keys (including symbols) we could just do:
```js
return [...Object.keys(object), ...Object.getOwnPropertySymbols(object)];
```

Or if we wanted to include both, I think we could:
```js
export function getAllEnumerableKeys(
  object: Object,
): Array<string | number | Symbol> {
  let keys = [];
  let current = object;
  while (current != null) {
    const currentKeys = [
      ...Object.keys(current),
      ...Object.getOwnPropertySymbols(current),
    ];
    const descriptors = Object.getOwnPropertyDescriptors(current);
    for (let key of currentKeys) {
      if (descriptors[key].enumerable) {
        keys.push(key);
      }
    }
    current = current.__proto__;
  }
  return keys;
}
```

Reply:
Yes @bvaughn you are right ,  I'm going to change it. Thank you

Reply:
Done :)